website/templates/base_generic.html

```diff
@@ -13,6 +13,8 @@
 		<!-- Add additional CSS in static file -->
 		{% load static %}
 		<link rel="stylesheet" href="{% static "css/styles.css" %}">
+		<script src="https://unpkg.com/jquery"></script>
+		<script src="https://unpkg.com/bootstrap"></script>
```

Comment:
I suggest using the official bootstrapcdn links for the script source to make sure the source are always reliable
```
<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
```